src/backup.sh

```diff
@@ -83,3 +83,13 @@ aws $aws_args s3 cp "$local_file" "$s3_uri"
 rm "$local_file"
 
 echo "Backup complete."
+
+if [ "$BACKUP_KEEP_DAYS" -ne 0 ]; then
+  ms=$((86400*BACKUP_KEEP_DAYS))
```

Comment:
Why `ms`? To me that suggests milliseconds, but looks like this is seconds.

Reply:
my bad.